Documentation/migration-scripts/python-scripts/fetching_and_auto_mapping_groups.py

```
@@ -0,0 +1,287 @@
+    )
+
+    config = ConfigFactory.parse_file(unified_conf)
+
+    return {
+        "url": config.get_string("metabase.url"),
+        "username": config.get_string("metabase.username"),
+        "password": config.get_string("metabase.password")
+    }
+
+def get_session(url, username, password):
+    res = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)